packages/siop-oid4vp/lib/__tests__/SdJwt.spec.ts

```diff
@@ -80,11 +84,26 @@ function getPresentationDefinition(): IPresentationDefinition {
   }
 }
 
+const sdJwtVcQuery: DcqlQuery = {
+  credentials: [
+    {
+      id: 'my_credential',
+      format: 'vc+sd-jwt',
+      meta: {
+        vct_values: ['https://high-assurance.com/StateBusinessLicense'],
+      },
+      claims: [{ path: ['license', 'number'] }, { path: ['user', 'name'] }],
+    },
+  ],
+}
+
+DcqlQuery.validate(sdJwtVcQuery)
+
 function getVCs(): OriginalVerifiableCredential[] {
   return [SD_JWT_VC]
 }
 
-describe.skip('RP and OP interaction should', () => {
+describe('RP and OP interaction should', () => {
```

Comment:
See earlier remark about enabling tests that have a TODO/FIXME

Reply:
Rolled it back to skip test